corehq/apps/hqwebapp/management/commands/build_requirejs.py

```diff
@@ -203,6 +203,13 @@ def _run_r_js(self, config, bootstrap_version=3):
             logger.info(f"{log_prefix}Copying {bootstrap_dir}/requirejs_config.js back to {os.path.relpath(dest)}")
             copyfile(filename, dest)
 
+        # Copy build files for later troubleshooting, since the B5 run of this function will overwrite the B3 files
+        for basename in ("build.js", "build.txt"):
+            src = self._staticfiles_path(basename)
+            dest = src.replace("build", f"build.b{bootstrap_version}")
```

Comment:
Is it possible `"build"` could appear more than once in `src`? Would it be a problem if it were replaced more than once?

Reply:
It's pretty unlikely and would likely make the copy fail, but I tightened it up a bit here: faef74a452d